src/br/com/vaasschool/model/CourseVisibility.java

```diff
@@ -1,6 +1,26 @@
 package br.com.vaasschool.model;
 
+import java.util.Arrays;
+
 public enum CourseVisibility {
-    PRIVATE,
-    PUBLIC
-}
+
+    PRIVATE ("PRIVADA"),
+    PUBLIC ("PÚBLICA");
+
+    private String displayName;
+
+    public String getDisplayName() {
+        return displayName;
+    }
+
+    CourseVisibility(String displayName) {
+        this.displayName = displayName;
+    }
+
+    public static CourseVisibility languageType(String value){
+        return Arrays.stream(CourseVisibility.values())
+                .filter(cv -> cv.getDisplayName().equalsIgnoreCase(value))
```

Comment:
Dá pra encapsular essa lógica lá no `CourseVisiability` e fazer algo como um `cv.is(value)`. Daí, lá dentro, a gente chama esse` getDisplayName().equalsIgnoreCase(value)`. É só uma dica, mas não precisa mudar não. :))